app/grandchallenge/components/views.py

```diff
@@ -182,3 +206,64 @@ def get_context_data(self, *args, **kwargs):
             }
         )
         return context
+
+
+class ObjectCreateMixin:

```

Comment:
`ObjectCreate` I think is too unspecific. I like the terminology of `DisplaySet` as this is what is used in DICOM, and much prefer it over `ArchiveItem`. I think using `DisplaySetCreate` would be too confusing as that has a different concrete class. These are sets of interface values right, so maybe `CIVSetCreate`? What do you think?

Reply:
Also, why is this separate from `MultipleCIVProcessingBaseView`? It's never used independently, right?

Reply:
We could then call `related_item_model` `civ_sets_field`.

Reply:
I like your wording, very clear, so `CIVSetCreate` and `civ_sets_related_manager` it is. Thanks!